homeassistant/components/light/qwikswitch.py

```diff
@@ -0,0 +1,37 @@
+"""
+Support for Qwikswitch Relays and Dimmers as HA Lights.
+
+See the main component for more info
+"""
+import logging
+from homeassistant.components.qwikswitch import QSToggleEntity, QSUSB

```

Comment:
This is dangerous. When this platform is loaded, it will copy the reference QSUSB is pointing at. Since you overwrite it in the main component, this platform will no longer point at the right constant.